src/journey-maps/angular/components/level-switch/level-switch-horizontal.ts

```diff
@@ -73,15 +73,24 @@ export class SbbLevelSwitchHorizontal implements OnInit, OnDestroy {
   toggleSideButtons(): void {
     this.showSideButtons = !this.showSideButtons;
     if (this.showSideButtons) {
-      this.startCountdown();
-      setTimeout(() => {
-        this._focusMatchingButton();
-      }, 0); // make sure side buttons are visible first
+      this.openSideButton();
     } else {
-      clearTimeout(this._countdownTimer);
+      this.closeSideButton();
     }
   }
 
+  closeSideButton(): void {
+    this.showSideButtons = false;
+    clearTimeout(this._countdownTimer);
+  }
+
+  openSideButton(): void {
```

Comment:
Und kann man hier nicht analog zum `closeSideButton` dann `this.showSideButtons = true;` machen?